2fa-api/handlers/login.js

```diff
@@ -0,0 +1,45 @@
+const db = require("../db");
+const twilio = require("../twilio");
+
+const twilioClient = twilio.twilioClient;
+
+const login = function (req, res) {
+  try {
+    const { email } = req.body;
+    const { password } = req.body;
+
+    db.connection.query(
+      `SELECT id, phone from Users WHERE email='${email}' and password='${password}'`,
+      (error, results, _) => {
+        if (error) {
+          console.log(error);
+          return res.status(500).json({ error });
```

Comment:
is this really a `500`? what should this be?

Reply:
Perhaps a 502 would be better here to indicate an issue with a downstream service, in this case the DB